### DISCUSSION TITLE
publish migrate add current date time to filename

### DISCUSSION THREAD

**Discussion by vahidalvandi** [spatie/laravel-settings#178] [Ideas]
hi . i have probleam with number in **2022_12_14_083155_create_settings_table.php** 
it is better that the title is allway fixed , becuase it run after older settings

**Comment by rubenvanassche** [comment]
Thanks for the fix!

**Comment by vahidalvandi** [comment]
please add this in v 2.6.0

**Reply by rubenvanassche** [reply]
Will be released in a sec
